README.md

```diff
@@ -115,4 +119,74 @@ This will generate a coverage.out file in the root directory of the repo which y
 
 ```
 $ go tool cover -html=coverage.out
-```
\ No newline at end of file
+```
+
+## Supported commands and sub-commands
+
+```
+    addressable
+      list        A list of all device services
+```   
+```
+    command
+      list        A list of device supported commands
+```   
+```
+    db
+      purge - Purges entire EdgeX Database. [USE WITH CAUTION]
+```
+
+```
+    device
+      add         Add devices
+      list        A list of all devices
+      rm          Removes device by name or ID
+```
+
+```
+    deviceservice
+      list        Lists existing devices services
```

Comment:
does addressable list and deviceservice list return the same thing?

Reply:
Good catch! I will fix the addressable list - helper msg and will update the list here

Reply:
separate issue created https://github.com/edgexfoundry-holding/edgex-cli/issues/240